packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+import Login from './Login';
+import LoginHeader from './LoginHeader';
+import LoginHeaderBrand from './LoginHeaderBrand';
+import LoginFooter from './LoginFooter';
+
+import LoginBox from './LoginBox';
+import LoginBoxHeader from './LoginBoxHeader';
+import LoginBoxBody from './LoginBoxBody';
+import LoginBoxFooter from './LoginBoxFooter';
+
+import { BackgroundImage } from '../BackgroundImage';
+import { Brand } from '../Brand';
+import { Text, TextContent } from '../Text';
+import { List } from '../List';
+
+const ListVariant = {
+  grid: 'grid',
+  inline: 'inline'
+};
+
+const propTypes = {
+  /** Anything that can be rendered inside of the LoginPage */
+  children: PropTypes.node,
+  /** Additional classes added to the LoginPage. */
+  className: PropTypes.string,
+  /** Attribute that specifies the URL of the brand image for the LoginPage */
+  brandImgSrc: PropTypes.string,
+  /** Attribute that specifies the alt text of the brand image for the LoginPage. */
+  brandImgAlt: PropTypes.string,
+  /** Attribute that specifies the URL of the background image for the LoginPage */
+  backgroundImgSrc: PropTypes.string,
+  /** Attribute that specifies the alt text of the background image for the LoginPage. */
+  backgroundImgAlt: PropTypes.string,
+  /** Dropdown component for the LoginBox Header of the LoginPage */
+  dropdown: PropTypes.node,
+  /** Content rendered inside of the Text Component of the LoginPage */
+  textContent: PropTypes.string,
+  /** Title for the LoginBox Header of the LoginPage */
+  title: PropTypes.string,
+  /** Items rendered inside of the List Component of the LoginPage */
+  listItems: PropTypes.node,
+  /** Text for the LoginBox Header of the LoginPage */
+  headerText: PropTypes.string,
```

Comment:
Instead of 3 props for header, can we have just one `loginSubtitle` perhaps, that accepts a node? Then they can do
```
const subtitle = <p>headerText <a href="headerUrl">headerUrlText</a></p>;
loginSubtitle={subtitle}
```